lib/Config.js

```diff
@@ -54,10 +55,16 @@ class Config extends EventEmitter {
             throw new Error(`could not parse config file: ${err.message}`);
         }
 
-        const { value: parsedConfig, err } = backbeatConfigJoi.validate(config);
-        if (err) {
-            throw new Error(`could not validate config file: ${err.message}`);
-        }
+        this._parseConfig(config);
+    }
+
+    /**
+     * Parses Backbeat's configuration
+     * @param {Object} config backbeat configuration
+     * @returns {undefined}
+     */
+    _parseConfig(config) {
+        const parsedConfig = joi.attempt(config, backbeatConfigJoi);
```

Comment:
should we not detect & throw an error if the config is not valid?

Reply:
The function throws an error when it fails